src/main/webapp/projectReview/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ProjectReview from './ProjectReview';
+import '../index.css';
+import ErrorHandler from '../components/ErrorHandler';
+
+ReactDOM.render(
+  <ErrorHandler>
+    <ProjectReview
+      issue = {projectReviewValues.issue}
+      searchUsersURL = {projectReviewValues.searchUsersURL}
+      projectKey = {projectReviewValues.projectKey}
+      projectUrl = {projectReviewValues.projectUrl}
+      addExtraPropUrl = {urls.saveExtraPropUrl}
+      isViewer = {projectReviewValues.isViewer}
+      serverURL = {projectReviewValues.serverURL}
```

Comment:
Some more instances of global urls that could be constants in the component instead of being passed in props.

Reply:
Indeed. I removed these constants. I belive that these indexes can be removed from react and wepack.config, but I would rather do this change in my next Pr.